Wrappers/Python/ccpi/viewer/utils/colormaps.py

```diff
@@ -465,6 +465,62 @@ def get_color_transfer_function(cls, cmap, color_range):
 
         return tf
 
+    @staticmethod
+    def get_lookup_table(cmap, color_range):
+
+        tf = vtk.vtkLookupTable()
+
+
+        if not cmap in _color_map_dict.keys():
+
+            try:
+                from matplotlib import cm
+                
+                colors = []
+                for x in range(0, 255):
+                    color = cm.get_cmap(cmap)(x)
+                    colors.append([color[0], color[1], color[2]])
+            except ImportError:
+                print("To use colormaps other than: ",
+                    "{}, please install matplotlib.".format(
+                        str(list(_color_map_dict.keys()))))
+                        
+        else:
+            try:
+                colors = _color_map_dict[cmap]
+            except KeyError as e:
+                raise KeyError("Colormap: {} could not be found. \
+                     Installing matplotlib might resolve this.".format(e))
+
+            try:
+                from matplotlib import cm
+                
+                colors = []
+                for x in range(0, 255):
+                    color = cm.get_cmap(cmap)(x)
+                    colors.append([color[0], color[1], color[2]])
+            except ImportError:
+                print("To use colormaps other than: ",
+                    "{}, please install matplotlib.".format(
+                        str(list(_color_map_dict.keys()))))
+                        
+        else:
+            colors = _color_map_dict[cmap]
+            
+        N = len(colors)
+        x = numpy.linspace(0, N, num=N)
+        scaling = 0.1
+        opacity = CILColorMaps.get_opacity_transfer_function(x, relu, color_range[0], color_range[1], scaling)
+        
+        for i, color in enumerate(colors):
+            # level = color_range[0] + \
+            #     (color_range[1] - color_range[0]) * i / (N-1)
+            tf.SetTableValue(i, color[0], color[1], color[2], opacity.GetValue(i))
+
+        return tf
+
+    
+
```

Comment:
```suggestion
```

I think you need to update your fork's master so that you don't keep requesting these changes

Reply:
ah, I must have made a local mess at some point!